packages/gatsby-cli/src/create-cli.js

```diff
@@ -23,6 +23,7 @@ const handlerP = fn => (...args) => {
 
 function buildLocalCommands(cli, isLocalSite) {
   const defaultHost = `localhost`
+  const defaultPort = process.env.PORT || `8000`
```

Comment:
```suggestion
  const defaultPort = parseInt(process.env.PORT, 10) || `8000`
```

Let's add a `parseInt`

Reply:
The `default` value below expects a a string though so why would we use `parseInt`?

Note that `8000` is a string too.

Reply:
Ah, right. I missed that. Fair enough.